crates/noirc_frontend/src/hir_def/function.rs

```diff
@@ -18,9 +18,10 @@ impl HirFunction {
         HirFunction(ExprId::empty_block_id())
     }
 
+    // <Note> Originally named unsafe_from_expr changed to unchecked_from_expr as per this issue https://github.com/noir-lang/noir/issues/812 </Note>
```

Comment:
We can get rid of this comment because this change will be reflected in the CHANGELOG.md

Reply:
Removed as per your request :D